bot/exts/evergreen/issues.py

```diff
@@ -92,15 +125,76 @@ async def issue(
             issue_url = json_data.get("html_url")
             links.append([icon_url, f"[{repository}] #{number} {json_data.get('title')}", issue_url])
 
-        # Issue/PR format: emoji to show if open/closed/merged, number and the title as a singular link.
-        description_list = ["{0} [{1}]({2})".format(*link) for link in links]
+        return links
+
+    @staticmethod
+    def get_embed(result: list, user: str = "python-discord", repository: str = "") -> discord.Embed:
+        """Get Response Embed."""
+        description_list = ["{0} [{1}]({2})".format(*link) for link in result]
         resp = discord.Embed(
             colour=Colours.bright_green,
             description='\n'.join(description_list)
         )
 
         resp.set_author(name="GitHub", url=f"https://github.com/{user}/{repository}")
-        await ctx.send(embed=resp)
+        return resp
+
+    @commands.command(aliases=("pr",))
+    async def issue(
+            self,
+            ctx: commands.Context,
+            numbers: commands.Greedy[int],
+            repository: str = "sir-lancebot",
+            user: str = "python-discord"
+    ) -> None:
+        """Command to retrieve issue(s) from a GitHub repository."""
+        if ctx.channel.category not in WHITELISTED_CATEGORIES or ctx.channel.category in WHITELISTED_CHANNELS:
+            return
+
+        result = await self.fetch_issues(set(numbers), repository, user)
+
+        if result == FetchIssueErrors.value_error:
+            await ctx.invoke(self.bot.get_command('help'), 'issue')
+
+        elif result == FetchIssueErrors.max_requests:
+            embed = discord.Embed(
+                title=random.choice(ERROR_REPLIES),
+                color=Colours.soft_red,
+                description=f"Too many issues/PRs! (maximum of {MAX_REQUESTS})"
+            )
+            await ctx.send(embed=embed)
+
+        elif isinstance(result, list):
+            # Issue/PR format: emoji to show if open/closed/merged, number and the title as a singular link.
+            resp = self.get_embed(result, user, repository)
+            await ctx.send(embed=resp)
+
+        elif isinstance(result, str):
+            await ctx.send(result)
+
+    @commands.Cog.listener()
+    async def on_message(self, message: discord.Message) -> None:
+        """Command to retrieve issue(s) from a GitHub repository using automatic linking if matching <repo>#<issue>."""
+        if message.channel.category not in WHITELISTED_CATEGORIES or message.channel.category in WHITELISTED_CHANNELS:
+            return
+
+        message_repo_issue_map = re.findall(fr".+?({self.repo_regex})#(\d+)", message.content)
+        links = []
+
+        if message_repo_issue_map:
+            for repo_issue in message_repo_issue_map:
+                if self.check_in_block(message, " ".join([*repo_issue])):
+                    continue
+                else:
```

Comment:
this could be better written using `not` 
```suggestion
                if not self.check_in_block(message, " ".join([*repo_issue])):
```